packages/core/src/lib/TwaManifest.ts

```diff
@@ -105,6 +105,7 @@ export class TwaManifest {
   generatorApp: string;
   webManifestUrl?: URL;
   fallbackType: FallbackType;
+  appsFlyer: boolean;
```

Comment:
Can we rather keep it open ended to include any analytics provider instead of just providing an option of Appsflyer? We can have a parameter like "addExistingAnalyticsProvider" and then next parameter can be a set of supported providers which could just appsflyer to begin with.

Reply:
Realized this has been taken care of in the future commits of including an array of plugins.